subcommand/webhook-cert-manager/command.go

```diff
@@ -251,6 +281,14 @@ func (c *Command) reconcileCertificates(ctx context.Context, clientset kubernete
 
 	certSecret.Data[corev1.TLSCertKey] = bundle.Cert
 	certSecret.Data[corev1.TLSPrivateKeyKey] = bundle.Key
+	certSecret.OwnerReferences = []metav1.OwnerReference{
```

Comment:
add a comment that this is here to update existing secrets that were created before we added ownerReference?

Reply:
done